src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +13,11 @@ public class Users {
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users
+            .stream()
+            .filter(user -> user.getName()==name)
```

Comment:
값을 꺼내는 대신에 `User`의 `matchName()`을 활용해보면 어떨까요?
```suggestion
            .filter(user -> user.matchName(name))
```